### ISSUE TITLE
Render causes error

### ISSUE THREAD

**Issue by foruzanco98** [vectorgraphics/asymptote#238]
When I compile using PDFLaTeX (TeX Live 2021), the following asy file is generated:
```
if(!settings.multipleView) settings.batchView=false;
settings.tex="pdflatex";
settings.inlinetex=true;
deletepreamble();
defaultfilename="inlinemovie3-1";
if(settings.render < 0) settings.render=4;
settings.outformat="";
settings.inlineimage=true;
settings.embed=true;
settings.toolbar=false;
viewportmargin=(2,2);

settings.render=4;
settings.prc=false;

import graph3;
import animate;
currentprojection=orthographic(1,-2,0.5);

animation A=animation("movie3");
int n=20;
for(int i=0; i < n; ++i) {
picture pic;
size3(pic,12cm,12cm,8cm);
real k=i/n*pi;
real f(pair z) {return 4cos(abs(z)-k)*exp(-abs(z)/6);}
draw(pic,surface(f,(-4pi,-4pi),(4pi,4pi),Spline),paleblue);
draw(pic,shift(i*6Z/n)*unitsphere,yellow);
A.add(pic);
}
label(A.pdf("autoplay,loop",delay=20,keep=!settings.inlinetex));
```
running Asymptote on this file yields the error:
```
D:\n524>asy inlinemovie3-1.asy
    shipout3(prefix,f,preview ? nativeformat() : format,
            ^
C:/texlive/2021/texmf-dist\asymptote/three.asy: 2878.13: runtime:       2 [main] asy 996 cygwin_exception::open_stackdumpfile: Dumping stack trace to asy.exe.stackdump
C:\texlive\2021\bin\win32\runscript.tlu:915: command failed with exit code 34816:
asy inlinemovie3-1.asy
```
removing the "settings.prc=false;" line and running again, one gets:
```
D:\n524>asy inlinemovie3-1.asy
    shipout3(prefix,f,preview ? nativeformat() : format,
            ^
C:/texlive/2021/texmf-dist\asymptote/three.asy: 2878.13: runtime:       1 [main] asy 1936 cygwin_exception::open_stackdumpfile: Dumping stack trace to asy.exe.stackdump
C:\texlive\2021\bin\win32\runscript.tlu:915: command failed with exit code 34816:
asy inlinemovie3-1.asy
```
restoring this line and just removing the two "settings.render=4;" lines, I get:
```
D:\n524>asy inlinemovie3-1.asy
    shipout3(prefix,f,preview ? nativeformat() : format,
            ^
C:/texlive/2021/texmf-dist\asymptote/three.asy: 2878.13: runtime:       2 [main] asy 9428 cygwin_exception::open_stackdumpfile: Dumping stack trace to asy.exe.stackdump
C:\texlive\2021\bin\win32\runscript.tlu:915: command failed with exit code 34816:
asy inlinemovie3-1.asy
```
However, removing all the three lines at once, makes Asymptote run without error. Albeit, the file "_movie3.pdf" seems to be an static 3D figure and "inlinemovie3.pdf" does not open in Adobe Acrobat Reader. What is going on?

**Comment by johncbowman** [comment]
What version of ghostscript are you using. Don't use ghostscript 9.54.0 (it is broken but a patch will be in 9.55). Use 9.53.3
instead.

**Comment by johncbowman** [comment]
Although I was unable to reproduce this issue, it is likely fixed as of Asymptote 2.73; otherwise please reopen this bug report.

**Comment by johncbowman** [comment]
See also https://github.com/vectorgraphics/asymptote/issues/364.
